### PR TITLE
Removed (going to fix, sign with a new key and re-upload)

### PR DESCRIPTION
**Pull request by mikebdp2**
```
title: "Open Compute Open System Firmware (OCP OSF) and its' importance"
abstract: "Let's figure out what is OCP Open System Firmware,
          see how it relates to the Open Source Firmware (OSF) - and why it is
          important for your company to have one running on your product."
```

### CODE REVIEW

File: blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+| Compute       | $13.25        | 83.1%             | $28.07       | 83.0%             | 16.2%          |
+| Storage       | $2.45         | 16.9%             | $5.73        | 17.0%             | 18.5%          |
+| **Total**         | **$15.70**        | **100.0%**            | **$33.80**       | **100.0%**            | **16.6%**          |
+
+*Source: IDC Worldwide Open Compute Project Compute and Storage Infrastructure Market Forecast, May 2020.*
+
+The OCP Open System Firmware project is aiming to allow system firmware like
+[BIOS][3] and [BMC][4] to be modified and their source code shared with the
+owners of OCP-certified hardware. Since March 2021, supporting Open System
+Firmware is mandatory for servers with OCP badging. The ecosystem partners have
```
Comment (pietrushnic) [inline blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md:47]:
```suggestion
Firmware is mandatory for servers with OCP badging. Some ecosystem partners have
```

File: blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+decided to implement such Open Source Firmware (OSF) as coreboot for their Open
+System Firmware architecture, since having the source code helps to adapt the
+firmware to the needs of even the most demanding clients by fixing the firmware
+bugs and expanding the device's feature set - at a reduced cost - which allows
+making the most out of both your existing and future products by extending their
+lifecycle and bringing them to the new previously unthought-of markets.
```
Comment (pietrushnic) [inline blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md:53]:
```suggestion
lifecycle and bringing them to the new previously unthought-of markets. Another reason is that full control over firmware helps in long-term hardware maintenance and removes the dependency on external companies, which at the time of the incident may or may not have resources to provide support.
```

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+fundamental features to the more advanced ones are depicted from left to right
+and 1st to 3rd are essential.
+
+1. Chip Initialization and Integration: the firmware components for the initial
+setup of the motherboard's key chips - like the chips themselves - are usually
+provided by the Silicon Vendors, sometimes in a binary-only form *(without the*
+*source code)* like Intel FSP and later AMD AGESA. We at [3mdeb][7] can audit the
+source code either available publicly or accessible to us thanks to our NDAs &
+close cooperative relationships with Silicon Vendors like Intel and AMD. When
+the sources aren't available, we can reverse-engineer a binary-only component to
```
Comment (miczyg1) [inline blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md:83]: Should we announce reverse-engineering of the blobs which license is against reverse engineering?

Comment (mikebdp2) [reply blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md:86]: @miczyg1 We don't announce there which blobs in particular we can reverse-engineer, so I think this is fine.

Comment (pietrushnic) [reply blog/content/post/2021-02-23-OSF_vs_OCP_OSF.md:86]: @miczyg1 @mikebdp2 this is fine.

**Review by mikebdp2 — COMMENTED** [review]
Anything else I could improve to get it merged - or it's okay already?

**Pull request closed without merging**